luminaire/model/model_utils.py

```diff
@@ -67,6 +69,7 @@ class LADHolidays(pandas.tseries.holiday.AbstractHolidayCalendar):
     USThanksgivingSunday = _H.Holiday('Thanksgiving Sunday', month=11, day=1,
                                    offset=[_DateOffset(weekday=_H.TH(4)), _DateOffset(days=3)])
 
+    Dec23 = _H.Holiday('December 23', month=12, day=23)
```

Comment:
Nit: There is no explicit name for this day?

Reply:
This is the day before Christmas eve. This is not a holiday but the holidays usually also impacts the neighboring days.

Reply:
yes, but do we want the variable name to be MonthDate like `Dec23`? I was proposing to use some user-friendly variable names for such thing like `DayBeforeChristmas `

Reply:
There are preexisting dates specified in the similar ways. I just followed the same convention. These dates just helping to catch any additional correlated patterns for being close to the holidays. So, they are not technically any special days  apart for the sake of the modeling purpose.